### PR TITLE
Report secret store failures

### PR DESCRIPTION
**Pull request by mlsteele**

### CODE REVIEW

File: go/libkb/osv_windows.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// +build windows
+
+package libkb
+
+// OSVersionAndBuild returns OS version, and build too on some platforms
+func OSVersionAndBuild() (string, string, error) {
+	productVersion, err := execToString("cmd", []string{"/c", "ver"})
```
Comment (zapu) [inline go/libkb/osv_windows.go]: I thought @heronhaye worked on something like this before, maybe it could be used?

Comment (heronhaye) [reply go/libkb/osv_windows.go]: Looks like we have it in the status package and in the install package. Could both depend on a libkb helper but doesn't really matter.

Comment (zapu) [reply go/libkb/osv_windows.go]: looks like it is used, sorry! but factored out to libkb?

Comment (heronhaye) [reply go/libkb/osv_windows.go]: Ah didn't see rest of PR. Looks fine to me

File: go/libkb/secret_store.go [hunk 2]
```diff
@@ -316,6 +317,11 @@ func (s *SecretStoreLocked) SetOptions(mctx MetaContext, options *SecretStoreOpt
 // store, retrieve, and then delete a secret with an arbitrary name. This should
 // be done before provisioning or logging in
 func PrimeSecretStore(mctx MetaContext, ss SecretStoreAll) (err error) {
+	defer func() {
+		if err != nil {
+			go reportPrimeSecretStoreFailure(mctx.BackgroundWithLogTags())
```
Comment (zapu) [inline go/libkb/secret_store.go]: it will also get called in few of the tests but this is probably fine

File: go/libkb/osv_windows.go (hunk 1 above)
Comment (mlsteele) [reply go/libkb/osv_windows.go]: In order to call it from libkb it had to move either into libkb or another package. Due to libkb->status->libkb

**Comment by mlsteele** [comment]
it passed CI ! 🎈

**Pull request merged**